### ISSUE TITLE
It doesn't train my voice. 

### ISSUE THREAD

**Issue by nothingness6** [rsxdalv/tts-generation-webui#239]
I followed the instruction below:
https://youtu.be/gYNCnltAHzY?si=4eAqdlQbezHR0Lpo

However, it doesn't apply to my voice, but generates a completely different voice. I have no clue what wrong is.

**Comment by rsxdalv** [comment]
Is this for voice cloning/replication?

**Comment by rsxdalv** [comment]
So assuming it's that - there are limitations. Are you using it for English or something else?
There are several factors - audio length, clarity, speech style - that determine if it will work.

**Comment by nothingness6** [comment]
> Is this for voice cloning/replication?

Yes, as I mentioned above, I just followed the instruction and it looks like just voice cloning.

**Comment by nothingness6** [comment]
> So assuming it's that - there are limitations. Are you using it for English or something else? There are several factors - audio length, clarity, speech style - that determine if it will work.

Ah, I see. I'll give it another shot. Thanks!
